source/deployment/on-boarding.rst

```diff
@@ -12,27 +12,27 @@ Getting Started Tasks
  
-These settings can also be set in the config.json file.  Please see our `configuration settings documenation <https://docs.mattermost.com/administration/config-settings.html>`__ for a full listing of all configuration settings.  
+These settings can also be set in the ``config.json`` file.  Please see our `configuration settings documenation <https://docs.mattermost.com/administration/config-settings.html>`__ for a full listing of all configuration settings.  
 
 3. Begin to onboard users by enabling account creation or by connecting an authentication service to assist with user provisioning.  
 
- - Users can be per-provisioned prior with migration and bulk loading data processes based on prior collaboration systems.  Please see our `migration guide <https://docs.mattermost.com/administration/migrating.html#migration-guide>`_ and `bulk loading documentation <https://docs.mattermost.com/deployment/bulk-loading.html>`_ for additional details.
- - `AD/LDAP authentication <https://docs.mattermost.com/deployment/sso-ldap.html#active-directory-ldap-setup-e10-e20>`_ and `SAML authentication <https://docs.mattermost.com/deployment/sso-saml.html>`_ are available for Enterprise Edition, providing identity management, single sign-on and automatic account provisioning.   
+ - Users can be pre-provisioned with migration and bulk loading data processes based on prior collaboration systems. Please see our `migration guide <https://docs.mattermost.com/administration/migrating.html#migration-guide>`_ and `bulk loading documentation <https://docs.mattermost.com/deployment/bulk-loading.html>`_ for additional details.
+ - `AD/LDAP authentication <https://docs.mattermost.com/deployment/sso-ldap.html#active-directory-ldap-setup-e10-e20>`_ and `SAML authentication <https://docs.mattermost.com/deployment/sso-saml.html>`_ are available for Enterprise Edition, providing identity management, single sign-on, and automatic account provisioning.   
 
 4. Enable integrations and plugins to connect your team's workflows and toolsets into Mattermost. 
 
  - To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, to go **System Console > Integrations**. More information on these integrations can be found `here <https://docs.mattermost.com/guides/integration.html>`_. 
  - To enable and manage plugins, go to **System Console > Plugins**.  Mattermost offers an `integration marketplace <https://integrations.mattermost.com/>`_ where you can see all available plugins available for upload. 
 
-If your organization  requires more structure and project management artifacts for the implementation of Mattermost, please see our `implementation plan <https://docs.mattermost.com/getting-started/implementation_plan.html>`__.
+If your organization requires more structure and project management artifacts for the implementation of Mattermost, please see our `implementation plan <https://docs.mattermost.com/getting-started/implementation_plan.html>`__.
```

Comment:
@wiersgallak, should we replace this with your roll out guide?

Reply:
@justinegeffen yes please!

Reply:
Done. :)